Source/monster.cpp

```diff
@@ -4528,11 +4528,7 @@ void M_FallenFear(Point position)
 		if (monster._mAi != AI_FALLEN || monster._mhitpoints >> 6 <= 0)
 			continue;
 
-		int intelligenceFactor = monster._mint;
-		if (monster.MType->mtype == MT_DFALLSP)
-			intelligenceFactor--;
-		int runDistance = std::max((7 - 2 * intelligenceFactor), 2);
-
+		int runDistance = std::max((8 - monster.MData->mLevel), 2);

```

Comment:
I am calling this explanatory variable. I prefer this above comments.
(since mgoalvar1 says nothing).

Reply:
Yeah we are slowly trying to rename things to have sensible names.